messenger/message-recorder.rst

```diff
@@ -0,0 +1,77 @@
+.. index::
+    single: Messenger; Record messages
+
+Events Recorder: Handle Events After CommandHandler Is Done
+===========================================================
+
+Let's take the example of an application that has a command (a CQRS message) named
```

Comment:
This is not `a CQRS message` but message. You suer messages too when you application use the CQS pattern for example.

Reply:
Indeed, CQRS doesn't necessitate any form of messages. Could "an imperative domain message" be an option? Or maybe a more thorough definition could be useful.